src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -270,18 +654,977 @@ module Tick : S = struct
+                                 ( LookupPattern Lookup
+                                 , (fun () ->
+                                     if_feature
+                                       ( LookupsPerRow 4
+                                       , (fun () -> gamma)
+                                       , fun () ->
+                                           field
+                                             "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                       )
+                                     * ( ( field
+                                             "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                         + beta )
+                                       * if_feature
+                                           ( LookupsPerRow 2
+                                           , (fun () ->
+                                               field
+                                                 "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                               + beta )
+                                           , fun () ->
+                                               field
+                                                 "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                           )
+                                       * if_feature
+                                           ( LookupsPerRow 3
+                                           , (fun () ->
+                                               field
+                                                 "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                               + beta )
+                                           , fun () ->
+                                               field
+                                                 "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                           )
+                                       * if_feature
+                                           ( LookupsPerRow 4
+                                           , (fun () ->
+                                               field
+                                                 "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                               + beta )
+                                           , fun () ->
+                                               field
+                                                 "0x0000000000000000000000000000000000000000000000000000000000000001"
+                                           ) )
+                                     * ( gamma
+                                       + ( joint_combiner
+                                           * cell (var (Witness 2, Curr))
```

Comment:
These correspond to the formula `curr_row(2*i+2)`and `curr_row(2*i+1`, the 3 seem ok to me